src/api/models/sql_models/benthicpit.py

```diff
@@ -115,65 +114,78 @@ class BenthicPITSUSQLModel(BaseSUSQLModel):
             {BenthicPITObsSQLModel.sql}
         )
         SELECT NULL AS id,
-        benthicpit_su.pseudosu_id,
+        uuid_generate_v4() AS pseudosu_id,
         {_su_fields},
-        {_agg_su_fields},
+        benthicpit_su.{_agg_su_fields},
```

Comment:
`_agg_su_fields` already has a table name prefixed to it, I've see this in a few spots?